Langs/0407.lang

```diff
@@ -69,14 +72,14 @@ Done=Fertig
+Architecture=Sollten Sie ein 32-Bit-Betriebssystem nutzen, erfordert dies eine 64-Bit-Neuinstallation von Windows 11. Ihre Daten müssen vorab gesichert werden. Eine 32-Bit-CPU erfordert einen Wechsel auf eine 64-Bit-CPU.
+Boot=Die Boot-Methode, die Ihr Computer zum Starten verwendet. Windows 11 benötigt den UEFI-Modus. Sollten Sie "Legacy" (traditioneller Startmodus) nutzen, kann dies oft in den BIOS/UEFI-Einstellungen umgestellt werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
+CPU Name=Der Prozessor, welcher in Ihrem Computer verbaut ist. Die Liste der zu Windows 11 kompatiblen Prozessoren kann sich noch ändern. Bei einer Inkompatibilität muss der Prozessor ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
+CPU Cores=Die Anzahl der Rechenkerne, die Ihrer CPU zur Verfügung stehen. Windows 11 benötigt mindestens eine 2-Kern-CPU. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
+CPU Speed=Die Geschwindigkeit, mit der Ihre CPU die Aufgaben abarbeitet. Windows 11 erfordert mindestens 1 GHz. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
 DirectX=Die Version des DirectX-Funktionslevels, den die Grafikkarte unterstützt. Dies ist unabhängig von der DirectX-Softwareversion. Es kann vorkommen, dass 'DirectX 12 API'-Karten diese Prüfung nicht bestehen.
 Disk Type=Der Partitionsstil Ihrer Festplatte oder SSD. Windows 11 benötigt eine GPT-Partition. Sollten Sie MBR anstatt GPT nutzen, können Sie dies mit dem MBR2GPT-Tool von Microsoft korrigieren.
-RAM=Die Größe des in Ihrem Computer installierten Arbeitsspeichers. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
+RAM=Die Größe des in Ihrem Computer installierten Arbeitsspeichers. Windows 11 erfordert mindestens 4 GB RAM. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
 Secure Boot=Secure Boot (sicherer Start) ist eine Funktion innerhalb des UEFI-BIOS. Für Windows 11 muss diese Funktion vorhanden sein. Wenn Secure Boot nicht erkannt wird, kann dies bei neueren PCs oder Laptops im UEFI-BIOS aktiviert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
-Storage=Der Speicherplatz Ihrer Festplatte oder SSD. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
+Storage=Der Speicherplatz Ihrer Festplatte oder SSD. Windows 11 erfordert mindestens 64 GB Speicherplatz. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
 TPM=Trusted Platform Module (TPM) ist ein Sicherheitsmodul des Computers, das von Windows 11 benötigt wird. Nicht immer ist ein separates TPM-Modul notwendig. Viele moderne CPUs enthalten diese Funktion. Die Aktivierung von TPM ist oft in den UEFI-BIOS-Einstellungen möglich. Im BIOS nennt sich diese Funktion "fTPM" (AMD) oder "PTT" (Intel). Schlagen Sie im Handbuch Ihres Motherboards nach.
```

Comment:
I read somewhere that manufacturer provide new fw updates to enable uefi and tpm on the devices.

Maybe we can add a hint on this in general and for all languages.

_cc: @rcmaehl, @micwoj92 _

Reply:
That's correct. Many manufacturers want to offer a BIOS update to generally activate TPM. 
In addition, some OEMs offer a TPM firmware update from 1.2 to 2.0. 

https://www.neowin.net/news/asus-begins-rolling-out-windows-11-compatible-firmware/
https://support.microsoft.com/en-us/windows/update-your-security-processor-tpm-firmware-94205cbc-a492-8d79-cc55-1ecd6b0a8022
https://www.danielengberg.com/hp-tpm-firmware-version-upgrade-from-1-2-to-2-0-tpm-configuration-utility/